src/redux/actions/passwordResetActions.js

```diff
@@ -0,0 +1,59 @@
+import axios from 'axios';
+import {
+  PASSWORD_REQUEST_LOADING, PASSWORD_REQUEST_SUCCESS, PASSWORD_REQUEST_FAILED, INITIAL_STATE,
+} from './types/resetPasswordType';
+
+export const requestLoading = () => ({
+  type: PASSWORD_REQUEST_LOADING,
+  payload: {
+    loading: true,
+  },
+});
+
+export const success = message => ({
+  type: PASSWORD_REQUEST_SUCCESS,
+  payload: {
+    loading: false,
+    success: true,
+    message,
+  },
+});
+
+export const failed = message => ({
+  type: PASSWORD_REQUEST_FAILED,
+  payload: {
+    loading: false,
+    message,
+  },
+});
+
+export const initialState = () => ({
```

Comment:
Why do you have initial state in an actions file?

Reply:
The action is dispatched 2 seconds after an error message is shown on the screen.
This was how I implemented the feedback you gave on clearing error messages from the screen.